configure.ac

```diff
@@ -83,7 +83,15 @@ PKG_CHECK_MODULES(ICU, icu-uc icu-io, [
   AC_SUBST([ICU_CFLAGS])
   AC_SUBST([ICU_LIBS])
 ],[
-with_icu=no
+  AC_CHECK_TOOL(ICU_CONFIG, icu-config, no)
+  AC_MSG_CHECKING([for ICU by using icu-config fallback])
+  if test "$ICU_CONFIG" != "no" && "$ICU_CONFIG" --version >/dev/null; then
+    with_icu=yes
+    AC_MSG_RESULT([yes])
+  else
+    AC_MSG_RESULT([no])
+    with_icu=no
+  fi
 ])
```

Comment:
You mean I actually have to _do something_ with the information detected. Silly me!